packages/ui/src/app/components/ErrorFallback.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+
+import { ButtonPrimary, ButtonsGroup } from '@/common/components/buttons'
+import { FailureIcon } from '@/common/components/icons'
+import { Modal, ModalBody, ModalFooterComponent, ModalHeader } from '@/common/components/Modal'
+
+interface FallbackProps {
+  error: Error
+  resetErrorBoundary: (...args: Array<unknown>) => void
+}
+
+export default function ErrorFallback({ error, resetErrorBoundary }: FallbackProps) {
+  return (
+    <Modal modalSize="m" onClose={resetErrorBoundary}>
+      <ModalHeader icon={<FailureIcon />} title="Something went wrong" onClick={resetErrorBoundary} />
```

Comment:
This is close to the [documentation](https://reactjs.org/docs/error-boundaries.html). However in our case most likely crashes hide in pages / tabs. From my view the goal wouldn't be a global modal  "Something went wrong" but replacing the specific component with a "Crashed" message and if possible a trace or instructions how triage and report.

Reply:
Yes I completely agree